reader/feed/handler.go

```diff
@@ -157,6 +157,27 @@ func (h *Handler) RefreshFeed(userID, feedID int64) error {
 	return nil
 }
 
+// MarkFeedAsRead marks all feed's entries as read since last check.
+func (h *Handler) MarkFeedAsRead(userID, feedID int64) error {
```

Comment:
You can probably move the content of this function to `ui/feed_mark_as_read.go`.